easybuild/easyconfigs/p/Python-bundle-PyPI/Python-bundle-PyPI-2023.06-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,494 @@
+easyblock = 'PythonBundle'
+
+name = 'Python-bundle-PyPI'
+version = '2023.06'
+
+homepage = 'https://python.org/'
+description = "Bundle of Python packages from PyPI"
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+toolchainopts = {'pic': True}
+
+builddependencies = [
+    ('UnZip', '6.0'),
+    ('Rust', '1.70.0'),  # required for setuptools-rust, which is needed for cryptography and bcrypt
+    ('pkgconf', '1.9.5'),
+    ('git', '2.41.0', '-nodocs'),  # required for pbr
+    ('setuptools-rust', '1.6.0'),  # required for cryptography and bcrypt
+    ('hatchling', '1.18.0'),
+    ('poetry', '1.5.1'),
+    ('scikit-build', '0.17.6'),
+]
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('binutils', '2.40'),
+    ('bzip2', '1.0.8'),  # required for bz2 package in Python stdlib
+    ('zlib', '1.2.13'),
+    ('libreadline', '8.2'),
+    ('ncurses', '6.4'),
+    ('SQLite', '3.42.0'),
+    ('XZ', '5.4.2'),
+    ('GMP', '6.2.1'),  # required for pycrypto
+    ('libffi', '3.4.4'),
+    ('OpenSSL', '1.1', '', SYSTEM),
+    ('cryptography', '41.0.1'),
+]
+
+exts_default_options = {
+    'download_dep_fail': True,
+    'sanity_pip_check': True,
+    'use_pip': True,
+}
+
+# order is important!
+# package versions updated 2023-06-26
+exts_list = [
+    ('blist', '1.3.6', {
+        'patches': [
+            'Python-3_9-blist-1.3.6-fix-undefined_symbol_PyObject_GC_IS_TRACKED.patch',
+            'Python-3.10-bist-1.3.6-compatibility.patch',
+            'Python-3.11-bist-1.3.6-compatibility.patch',
+        ],
```

Comment:
Minor comment: I suggest to concatenate this 3 patched into a single patch file to keep things tidy.

Reply:
this comment can be ignored

Reply:
I am not sure about this. I followed the same approach as done in the previous Python easyconfigs, where we added the patches separately. For example: https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/p/Python/Python-3.10.8-GCCcore-12.2.0.eb#L57
In principle I like the idea of concatenating the patches, but one drawback would be that this would change the headers of the individual patches.